.gitlab-ci.yml

```diff
@@ -0,0 +1,28 @@
+image: golang:1.8
+
+stages:
+  - test
+  - build
+
+before_script:
+  - mkdir ${CI_PROJECT_DIR}/build
+  - mkdir -p ${GOPATH}/src/github.com/google/
+  - ln -s $(pwd) ${GOPATH}/src/github.com/google/mtail
+  - cd ${GOPATH}/src/github.com/google/mtail
+
+test:
+  stage: test
+  allow_failure: true
+  script:
+    - make test
+
+build:
+  stage: build
+  script:
+    - make install
+    - go build -o ${CI_PROJECT_DIR}/build/mtail
```

Comment:
I don't understand how this used, why not copy from $GOPATH/bin/mtail ?  Is this really masking a bug in the install?